### ISSUE TITLE
[1.16.X] Astral Sorcery

### ISSUE THREAD

**Issue by Niconoko** [CyanideX/Unity-Modded#58]
- Astral Sorcery

https://www.curseforge.com/minecraft/mc-mods/astral-sorcery

**Comment by CyanideX** [comment]
I removed Mana and Artiface from the suggestion after looking at this: 

![image](https://user-images.githubusercontent.com/3106700/106788534-ebb04700-6605-11eb-9693-4281085d2aa3.png)

I don't even think I can make this fit within Unity let alone make it look like it belongs in Minecraft. 😝 

Also, please create separate issues for each mod suggestion next time; it makes it a lot easier to manage each request, especially if I wish to close one.

**Comment by scaredofgirls** [comment]
I would be delighted to see this for 1.15.2 as well!

https://www.curseforge.com/minecraft/mc-mods/astral-sorcery/files/all?filter-game-version=1738749986%3a68722

**Comment by CyanideX** [comment]
I agree that the mod needs help but it's going to be a lot of work. A LOT. I cannot guarantee that I will be able to get around to this so don't get your hopes up. If I do get around to it, it wouldn't be for 1.15.
